### DISCUSSION TITLE
Onboarding PRs never got created

### DISCUSSION THREAD

**Discussion by sgandla1-godaddy** [renovatebot/renovate#20779] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

Installed Renovate app and selected two repositories. I never saw onboarding PRs getting created, There are no jobs running in the renovate dashboard and no logs to look at. The repositories are private

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Sorry, we are having some problems with AWS right now. The machine types we use are unavailable, so some reconfiguration is needed. Your repos will be onboarded once we have sufficient capacity.

**Reply by rarkins** [reply]
Now resolved
